Sources/Passage/PassageAuth.swift

```diff
@@ -190,6 +190,34 @@ public class PassageAuth {
         return authResult
     }
     
+    /// Activates a One-Time Passcode
+    ///
+    /// Upon successful activation the tokens will be stored in the tokenStore on the instance.
+    ///
+    /// - Parameter otp: string - The OTP provided by your user
+    /// - Parameter otpId: string - The OTP id returned from login or register method
+    /// - Returns: ``AuthResult`` The AuthResult object contains an authentication token (JWT) and redirect URL. The auth token should be used on all subsequent authenticated requests to the app. The redirect URL specifies the route that users should be redirected to after completed registration or login
```

Comment:
Is there any way to make this reusable? Seems to be used a bunch of places and would it nice if we could just store it as a string value - I don't know how you'd do that in this syntax though.

Reply:
Not that I'm aware of. But I am hoping to spend some time cleaning up inline docs when we can budget time for iOS work 🤞